### PR TITLE
text-align: left; for phones issue #49

### PR DESCRIPTION
**Pull request by Fransenson**
Added media query to align text left on phones. max-width: 767px should
cover most phones in portrait and landscape.

### CODE REVIEW

**Comment by a-kile** [comment]
Nice! :D 🌮

File: issues/css/index.css [hunk 1]
```diff
@@ -264,6 +264,13 @@ section .issue .body p {
   padding: 0 10px;
 }
 
+@media (max-width: 767px) {
+  section .issue .body p {
+    text-align: left;
+    padding: 0 10px;
```
Comment (vinaypuppal) [inline issues/css/index.css]: @Fransenson overriding `padding` is not necessary here. Fix this, we can merge.

**Comment by Fransenson** [comment]
@vinaypuppal I removed the padding. :)

**Comment by vinaypuppal** [comment]
@Fransenson Thank you

**Pull request merged**